# Write a program to reverse any string in Python

In [2]:
# write your code here...
s = str(input("Enter a string"))
print("Your enter string : "+s)
print("The reversed string : "+s[::-1])

Your enter string : example string 1234
The reversed string : 4321 gnirts elpmaxe


#### Input : Enter a stringexample string 1234 
#### Output : The reversed string is :   4321 gnirts elpmaxe

# Write a program to remove any given character from a string

In [1]:
# write your code here...
s = str(input("Enter any string"))
print("Your enter string : "+s)
r = str(input("Enter the character to be removed"))
print("The character to be removed : "+r)
ns = ""

for i in s :
    if i == r :
        ns = s.replace(r,"")
        continue
if ns == "" :
    print('''There is no "'''+r+'''" can be removed''')
else :
    print("The result after removed character : "+ns)

Your enter string : examplez123
The character to be removed : z
The result after removed character : example123


#### User Input1: Enter any string : examplez123
#### User Input2: Enter the character to be removed : z
#### Output: example123

# Write a program to check a string is palindrome or not

In [3]:
# write your code here...
def CfP(c) :
    return c == c[::-1]

s = str(input("Enter any string"))
print("Your enter string : "+s)

if CfP(s) :
    print("Is palindrome ? Yes")
else :
    print("Is palindrome ? No")

Your enter string : madam
Is palindrome ? Yes


#### Input: Enter any string : madam
#### Output: Yes

# Write a program to check if a given character is digit or not

In [5]:
# Write your code here...
c = input("Enter your own character")
print("Your enter character : "+c)

if c.isdigit() :
    print("The given character "+c+" is a digit")
else :
    print('''The given character "'''+c+'''" is NOT a digit''')

Your enter character : 76
The given character 76 is a digit


#### Input: Please Enter Your Own Character : 76
#### Output: The Given Character  76 is a Digit

# Write a program to check whether the given integer is a multiple of 7

In [6]:
# write your code here...
num = input("Enter an integer")
print("Your enter integer : "+num)

if int(num)%7 == 0 :
    print(num+" is a multiple of 7")
else :
    print(num+" is NOT a multiple of 7")


Your enter integer : 21
21 is a multiple of 7


#### Input:Enter an integer:21
#### Output: 21 is a multiple of 7

# Write a program to check whether the given integer is a multiple of both 7 and 11

In [7]:
# write your code here...
num = input("Enter an integer")
print("Your enter integer : "+num)

if int(num)%7 == 0 and int(num)%11 == 0 :
    print(num+" is a multiple of both 7 and 11")
else :
    print(num+" is NOT a multiple of both 7 and 11")

Your enter integer : 77
77 is a multiple of both 7 and 11


#### Input:Enter an integer: 77
#### Output: 77 is a multiple of both 7 and 11

# Write a program to reverse an integer in Python

In [8]:
# write your code here...
i = str(input("Enter an integer"))
print("Your enter integer : "+i)
print("Reversed Integer : "+i[::-1])

Your enter integer : 432424593
Reversed Integer : 395424234


#### Input:Enter an integer: 432424593
#### Output: Reversed Integer:  395424234